test/examples/src/update_doc.py

```diff
@@ -89,6 +89,6 @@
 
 except ApiException as ae:
```

Comment:
Could you remove code from L84-L86? This has been done in the JS and Go examples as well.